docs/reference/image_plugins.rst

```diff
@@ -0,0 +1,59 @@
+============================================
+Registering external image types via plugins
+============================================
+
+
+Usage
+~~~~~
+
+Toga can be extended, via plugins, to understand externally defined image types, gaining
+the ability to convert them to and from its own :any:`toga.Image` class.
+
+A plugin defining how to handle an image format must define three things:
+
+* ``image_class``
+
+  The class representing an image that we want to tell Toga about. For our example, it
+  would be assigned ``MyImage``. (You will, of course, have to import the relevant
```

Comment:
This reference to "MyImage" precedes the actual introduction of the example.

Reply:
Whoops! I rearranged this whole page after reintegrating the plugin package, and I missed that one.